experimental/devices/mfrc522/mfrc522.go

```diff
@@ -432,7 +436,7 @@ func (r *Dev) ReadBlock(sector int, block int) ([]byte, error) {
 // 	block - the block within the sector to write into.
 // 	data - 16 bytes if data to write
 // 	key - the key used to authenticate the card - depends on the used auth method.
-func (r *Dev) WriteBlock(auth byte, sector int, block int, data [16]byte, key []byte) (err error) {
+func (r *Dev) WriteBlock(auth byte, sector int, block int, data [16]byte, key [6]byte) (err error) {
```

Comment:
I'm ambivalent about this, because this copy the whole array at the invocation.

But copying a slice is copying 3 pointers, so in practice it's 12 or 24 bytes anyway. ¯\_(ツ)_/¯

Reply:
Here I wanted to ensure in the signature of the function that the user must send exactly 16 bytes. It might be a good idea to revert to slices and capture things at runtime, but from my perspective it's better to handle that at the compile time.

Perhaps I should pass a pointer to an array?